config/samples/metallb.io_v1alpha1_addresspool.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample1
+  namespace: metallb-system
+spec:
+  address-pool:
+    name: default
+    protocol: layer2
+    addresses:
+    - 172.18.0.100-172.18.0.200
+
+---
+
+apiVersion: metallb.io/v1alpha1
+kind: AddressPool
+metadata:
+  name: addresspool-sample2
+  namespace: metallb-system
+spec:
+  address-pool:
+    name: gold
+    protocol: layer2
+    addresses:
+    - 172.20.0.100-172.20.0.200
+    auto-assign: false
```

Comment:
This should probably be `autoAssign: false` to match kubernetes style

Reply:
I was trying to match the same configuration that the metallb is using today ?

Reply:
we don't have to match it, this is our chance to clean it up :)